R2Northstar/NorthstarMods
# Fix INVALID_REF errors caused by mods

INVALID_REF errors were being caused by trying to get the item image for weapon skins. 

If a mod (moreskins) added new skins, the client equipped one, and then the mod is disabled, the client now has bad persistence.
This is mostly handled fine by the game, except for weapon skins. This PR prevents the crash, and resets the bad persistence to default.

## Testing:

- Install [MoreSkins](https://northstar.thunderstore.io/package/Creamy_jpg/MoreSkinsMinus/) and equip a weapon warpaint that does not exist in vanilla (example being masterwork for the r201)
- Make sure your persistence has saved (close the game and reopen it, validate that the skin is still there)
- Delete/disable MoreSkins
- Go to the loadout, the weapon should now have Factory Issue equipped

### Note: This PR introduces two vanilla files, `menu_pilot_loadouts_shared.nut` and `menu_edit_pilot_loadouts.nut` to `Northstar.Client/mod/scripts/vscripts/ui`, they will need to be committed as Respawn.
